platform/darwin/src/MGLPolyline.mm

```diff
@@ -52,6 +53,27 @@ - (BOOL)isEqual:(id)other {
     return self == other || ([other isKindOfClass:[MGLPolyline class]] && [super isEqual:other]);
 }
 
+- (CLLocationCoordinate2D)coordinate {
+    mbgl::Polygon<double> polyline;
```

Comment:
Specifically, [`turf-along`](https://github.com/Turfjs/turf/blob/142e137ce0c758e2825a260ab32b24db0aa19439/packages/turf-along/index.js) was ported to Swift as [`coordinate(at:fromStartOf:)`](https://github.com/mapbox/mapbox-navigation-ios/blob/0acfa7f3719418af138b49d970be586092fea5ea/MapboxCoreNavigation/Geometry.swift#L228). (Note that Geometry.swift relies on [this overloaded operator](https://github.com/mapbox/mapbox-navigation-ios/blob/0acfa7f3719418af138b49d970be586092fea5ea/MapboxCoreNavigation/Geometry.swift#L63-L67), which can be difficult to spot sometimes.)